rinja/benches/escape.rs

```diff
@@ -13,7 +13,7 @@ fn escaping(b: &mut criterion::Bencher<'_>) {
     Lorem ipsum dolor sit amet, consectetur adipiscing elit. Mauris consequat tellus sit
     amet ornare fermentum. Etiam nec erat ante. In at metus a orci mollis scelerisque.
     Sed eget ultrices turpis, at sollicitudin erat. Integer hendrerit nec magna quis
-    venenatis. Vivamus non dolor hendrerit, vulputate velit sed, varius nunc. Quisque
+    venenatis. Vivamus non dolor hendrerit, vulputate velit sed, various nunc. Quisque
```

Comment:
Are we fixing latin too? XD

Reply:
Yea, in all other cases "various" would be right. I hope Cicero wouldn't mind that I changed his text. :)

Reply:
Wouldn't it be better in this case to move this text out of source code?

Reply:
Probably. It's used in two places anyway.

Reply:
Moved.